fix: Inacccurate info About Private Relay (#2707)
privacyguides/privacyguides.org: docs/os/ios-overview.md

A paid **iCloud+** subscription (with any iCloud storage plan) comes with some privacy-protecting functionality. While these may provide adequate service for current iCloud customers, we wouldn't recommend purchasing an iCloud+ plan over a [VPN](../vpn.md) and [standalone email aliasing service](../email-aliasing.md) just for these features alone.

> [**Private Relay**](https://www.apple.com/legal/privacy/data/en/icloud-relay/) is a proxy service which relays your Safari traffic and all unencrypted traffic on your device as well as your DNS queries through two servers: one owned by Apple and one owned by a third-party provider (including Akamai, Cloudflare, and Fastly). In theory this should prevent any single provider in the chain—including Apple—from having full visibility into which websites you visit while connected. Unlike a VPN, Private Relay does not protect traffic that's already encrypted.

Reviewer: ```suggestion
[**Private Relay**](https://apple.com/legal/privacy/data/en/icloud-relay) is a proxy service which relays all of your Safari traffic, your DNS queries, and unencrypted traffic on your device through two servers: one owned by Apple and one owned by a third-party provider (including Akamai, Cloudflare, and Fastly). In theory this should prevent any single provider in the chain—including Apple—from having full visibility into which websites you visit while connected. Unlike a VPN, Private Relay does not protect traffic that's already encrypted.
```

> [**Private Relay**](https://www.apple.com/legal/privacy/data/en/icloud-relay/) is a proxy service which relays your Safari traffic and all unencrypted traffic on your device as well as your DNS queries through two servers: one owned by Apple and one owned by a third-party provider (including Akamai, Cloudflare, and Fastly). In theory this should prevent any single provider in the chain—including Apple—from having full visibility into which websites you visit while connected. Unlike a full VPN, Private Relay does not protect traffic that's already encrypted.

Reviewer: ```suggestion
[**Private Relay**](https://apple.com/legal/privacy/data/en/icloud-relay) is a proxy service which relays all of your Safari traffic, unencrypted traffic on your device, and your DNS queries through two servers: one owned by Apple and one owned by a third-party provider (including Akamai, Cloudflare, and Fastly). In theory this should prevent any single provider in the chain—including Apple—from having full visibility into which websites you visit while connected. Unlike a full VPN, Private Relay does not protect traffic that's already encrypted.
```
Suggestions to clean the Private Relay URL and arrange the relayed items into a list
